### PR TITLE
Update dependencies that we can use this module with aws provider 3.x

### PR DESCRIPTION
**Pull request by dmitrijn**
## what
* Update module dependencies for aws provider 3.x

## why
* Update last conflicted dependency with aws provider 3.x (https://github.com/cloudposse/terraform-aws-alb-target-group-cloudwatch-sns-alarms)

## references
* https://github.com/cloudposse/terraform-aws-alb-target-group-cloudwatch-sns-alarms/pull/25

### CODE REVIEW

**Comment by joe-niland** [comment]
Hi @dmitrijn can you please run `make readme`?

**Comment by dmitrijn** [comment]
@joe-niland @jamengual can you run tests one more time?

**Comment by dmitrijn** [comment]
@joe-niland @jamengual could you suggest what to do with failed tests? as i see depended modules supports context and adds attributes in end of line, not that is expected by tests. what we can do?

**Comment by joe-niland** [comment]
@dmitrijn I haven't had a chance to look in detail but it looks like the issue is the labelling of the cloudwatch alarms.

The order of attributes added at the end of the label id has changed.

Will try to look at this a bit later.

**Comment by dmitrijn** [comment]
@joe-niland could you rerun terratest?

**Comment by dmitrijn** [comment]
@joe-niland pls, rerun tests one more time 🙂

**Comment by dmitrijn** [comment]
@joe-niland hurrah 🙂 looks like all is fine now

File: variables.tf [hunk 1]
```diff
@@ -233,7 +198,7 @@ variable "mount_points" {
   default     = []
 }
 
-variable "environment" {
+variable "list_environment" {
```
Comment (joe-niland) [inline variables.tf:201]: I think this should be called container_environment and the one below called map_container_environment

Comment (dmitrijn) [reply variables.tf]: @joe-niland fixed

**Review by joe-niland — APPROVED** [review]

**Comment by joe-niland** [comment]
Merge @dmitrijn thanks for your work and your patience! 🙂

**Pull request merged**